app/scripts/controllers/transactions/index.js

```diff
@@ -568,7 +568,6 @@ export default class TransactionController extends EventEmitter {
     if (!this._checkIfTxStatusIsUnapproved(txId)) {
       return;
     }
-
```

Comment:
This change is unrelated and can probably be reverted.

Reply:
fixed here [4f107a9](https://github.com/MetaMask/metamask-extension/pull/13646/commits/4f107a903a983d58be8f2b905927939894804c4e)